modules/aws_base/input_schema.aws_base.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$id": "http://localhost:7800/modules/aws_base/input_schema.aws-base.json",
```

Comment:
how can we merge this without having a reference to localhost?
do we even need this to be a URL?
The existing id are actually not valid URL
```
curl https://app.runx.dev/modules/aws-base
curl: (6) Could not resolve host: app.runx.dev
```